### ISSUE TITLE
Change length of RepeatVector

### ISSUE THREAD

**Issue by maxbry** [fchollet/keras#2624]
My model uses repeat vector to implement sequence to sequence:

    seq = Input(shape=(None,), dtype='int32')
    gru = GRU(hidden_size, dropout_W=dropout, dropout_U=dropout, return_sequences=False)(seq)
    rep_layer = RepeatVector(2)
    rep = rep_layer(gru)
    gru2 = GRU(hidden_size, dropout_W=dropout, dropout_U=dropout, return_sequences=False)(rep)
    out = TimeDistributed(Dense(output_dim=input_size-1, activation='softmax')
    
    [...]
    rep_layer.n = 3

What I want to do is change the `n` value of the `RepeatVector` layer in order to create longer sequence. When I do that, the length of the generated sequence does not change as expected, it stays the same. Do I need to recompile the model first?

**Comment by suraj-deshmukh** [comment]
what do u mean longer sequence??? RepeatVector layer just does the job of repeating the input n times.

**Comment by maxbry** [comment]
Yes, and after some point in training I want to change that `n` and set a higher value to repeat the input more often. But when I changed the value of `n` in my experiments, the input was not repeated more often, the number of repetitions stayed the same.

**Comment by lemuriandezapada** [comment]
yes, because keras makes the entire graph and sets all the dimensions
what you need to do is to make a new, identical network with the new value of n, compile it and copy over the weights
